packages/react/src/formElements/MdCheckboxGroup.tsx

```diff
@@ -55,6 +56,7 @@ const MdCheckboxGroup: React.FunctionComponent<MdCheckboxGroupProps> = ({
 
   const optionsClassNames = classnames('md-checkboxgroup__options', {
     'md-checkboxgroup__options--vertical': direction === 'vertical',
+    'md-checkboxgroup__options--grid': direction === 'grid',
```

Comment:
Nå som det er en ny "variabel" kan denne legges til i README eksemepelet for checkboxgroup?
`packages/css/src/formElements/checkboxgroup/README.md`